src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -268,27 +268,27 @@ public void visitToken(DetailAST ast) {
     }
 
     /**
-     * Some javadoc.
-     * @param modifierNode Some javadoc.
-     * @return Some javadoc.
+     * Checks whether a given modifier node has an annotation.
+     * @param modifierNode modifier node.
+     * @return true if a given modifier node has an annotation.
      */
     private static boolean hasAnnotations(DetailAST modifierNode) {
         return modifierNode != null && modifierNode.findFirstToken(TokenTypes.ANNOTATION) != null;
     }
 
     /**
-     * Some javadoc.
-     * @param modifierNode Some javadoc.
-     * @return Some javadoc.
+     * Returns an annotation level.
```

Comment:
`annotation level` isn't very clear. Can we expand javadoc to say it is the column position or indentation?